.github/workflows/android_builds.yml

```diff
@@ -68,6 +62,12 @@ jobs:
       - name: Extract pre-built Android Swappy Frame Pacing Library
         run: 7za x -y swappy/godot-swappy.7z -o${{github.workspace}}/thirdparty/swappy-frame-pacing
 
+      - name: Restore Godot build cache
+        uses: ./.github/actions/godot-cache-restore
+        with:
+          cache-name: ${{ matrix.cache-name }}
+        continue-on-error: true
```

Comment:
I just moved the cache restore and save as close as possible to the compiling step to minimal the time where the cache is locked.